packages/kit/src/core/sync/create_manifest_data/index.spec.js

```diff
@@ -214,37 +210,41 @@ test('encodes invalid characters', () => {
 	);
 });
 
-test('sorts routes correctly', () => {
-	const { routes } = create('samples/sorting');
-
-	assert.equal(
-		routes.map((p) => p.id),
-		[
-			'',
-			'(foo)',
-			'(foo)/(bar)',
-			'about',
-			'foo',
-			'post',
-			'post/bar',
-			'post/foo',
-			'post/f[yy].json',
-			'post/f[zz]',
-			'post/f[xx]',
-			'post/f[yy]',
-			'foo/[bar]',
-			'post/[id]',
-			'[endpoint]',
-			'[wildcard]',
-			'[...rest]/deep/[...deep_rest]/xyz',
-			'[...rest]/deep/[...deep_rest]',
-			'[...rest]/abc',
-			'[...rest]/deep',
-			'(foo)/(bar)/[...all]',
-			'[...anotherrest]',
-			'[...rest]'
-		]
+test.only('sorts routes correctly', () => {
```

Comment:
One of the things ESLint was doing before it was removed was to prevent things like this from being pushed. uvu doesn't support conditionally disallowing `only`/`skip`, so I had gone another route. If we don't want ESLint, we need to find another way to prevent this.